json/exp.json

```diff
@@ -9,8 +13,19 @@
         "verilator",
         "xgboost"
     ],
-    "experiment": "slurm_experiment",
-    "docker_home": "/<path>/<to>/slurm_home",
+    "_experiment": "experiment name",
```

Comment:
rephrase: "The name of the experiment. scarab-infra will create a new folder named <experiment> containing all files needed to execute the simulation in the FIXME directory. The <experiment> folder will also contain all generated simulation statistics/outputs."